src/mito2/src/sst/file.rs

```diff
@@ -109,6 +109,18 @@ pub struct FileMeta {
     pub available_indexes: SmallVec<[IndexType; 4]>,
     /// Size of the index file.
     pub index_file_size: u64,
+    /// Number of rows in the file.
+    ///
+    /// For historical reasons, this field might be missing in old files. Thus
+    /// the default value `0` doesn't means the file doesn't contains any rows,
+    /// but instead means the number of rows is unknown.
+    pub num_rows: u64,
```

Comment:
Question: How about to use `Option<u64>`?

Reply:
No need. Here `0` and `None` mean the same thing.